stages/org.osbuild.gcp.guest-agent.conf

```diff
@@ -0,0 +1,259 @@
+#!/usr/bin/python3
+"""
+Create a google-compute-engine config in /etc/default/instance_configs.cfg.distro
+
+Configures the guest environment behaviors without overwriting instance specific
+parameters in /etc/default/instance_configs.cfg.
```

Comment:
The comment should be updated a bit:

- The stage creates a "GCP guest-agent" config.
- Not only the `.distro` config can be created.